content/docs/administration/troubleshoot/databasecomponentunreadyreplicas.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "DatabaseComponentUnreadyReplicas"
+description: ""
+summary: ""
+date: 2025-06-05T13:52:09+03:00
+lastmod: 2025-06-05T13:52:09+03:00
+draft: false
+weight: 100
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Meaning
+
+Database component has unready replicas.
+
+{{< details "Full context" open >}}
+Database resource has a component with replicas which were declared to be unready.
+Database components impacted by this alert are Transaction Engines (TEs) and Storage Managers (SMs)
+For example, it is expected for a database to have 2 TE replicas, but it has less than that for a noticeable period of time.
+
+On rare occasions, there may be more replicas than it should and system did not clean it up.
```

Comment:
I would rephrase this as "..., there may be more replicas than request and the system not clean them up."